app/src/pages/inside/filtersPage/filtersPage.jsx

```diff
@@ -159,6 +164,9 @@ export class FiltersPage extends Component {
     this.props.tracking.trackEvent(FILTERS_PAGE_EVENTS.CLICK_ADD_FILTER_BTN);
   };
 
+  renderNoItemMessage = () =>
+    _renderNoItemMessage(this.props.intl, this.props.filter, messages.filtersNotFound);
```

Comment:
`NoFiltersFound` with corresponding props. Also you can put `this.props.intl.formatMessage` instead of `this.props.intl`

Reply:
fixed